fzf-url.sh

```diff
@@ -35,13 +35,13 @@ else
     content="$(tmux capture-pane -J -p -S -"$limit")"
 fi
 
-mapfile -t urls < <(echo "$content" |grep -oE '(https?|ftp|file):/?//[-A-Za-z0-9+&@#/%?=~_|!:,.;]*[-A-Za-z0-9+&@#/%=~_|]')
-mapfile -t wwws < <(echo "$content" |grep -oE '(http?s://)?www\.[a-zA-Z](-?[a-zA-Z0-9])+\.[a-zA-Z]{2,}(/\S+)*' | grep -vE '^https?://' |sed 's/^\(.*\)$/http:\/\/\1/')
-mapfile -t ips  < <(echo "$content" |grep -oE '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}(:[0-9]{1,5})?(/\S+)*' |sed 's/^\(.*\)$/http:\/\/\1/')
-mapfile -t gits < <(echo "$content" |grep -oE '(ssh://)?git@\S*' | sed 's/:/\//g' | sed 's/^\(ssh\/\/\/\)\{0,1\}git@\(.*\)$/https:\/\/\2/')
+urls=$(echo "$content" |grep -oE '(https?|ftp|file):/?//[-A-Za-z0-9+&@#/%?=~_|!:,.;]*[-A-Za-z0-9+&@#/%=~_|]')
```

Comment:
How about adding some comments so that others will known why we don't use `mapfile` here?

Reply:
I do not think other people will be questioning about `mapfile` if they won't see any usage of it. `shellcheck` also won't complain or suggest using `mapfile` in this PR.

What I've done to get rid of `mapfile`: I just assign the whole text output of the `grep` command to the variable. 
After that the whole string gets split into multiple items at whitespace character by accessing it as `"${urls[@]}"`. Luckily we can afford ourselves to rely on it (`@`), because there are no whitespaces in urls, and the single string with a bunch of urls will be split correctly, as expected.

Feel free to add a note about avoiding `mapfile`, if you think that the explanation is needed. To me it's not needed and `git blame/log` can answer the question to others if someone really wants to understand that.

Reply:
Agreed.